Lib/test/test_logging.py

```diff
@@ -995,7 +995,7 @@ def test_basic(self):
         self.assertEqual(mailfrom, 'me')
         self.assertEqual(rcpttos, ['you'])
         self.assertIn('\nSubject: Log\n', data)
-        self.assertTrue(data.endswith('\n\nHello \u2713'))
+        self.assertTrue(data.endswith('\n\nSGVsbG8g4pyTCg=='))
```

Comment:
This is problematic.  I'm guessing the test server needs to be told to support utf8 so the test itself doesn't change, but I haven't looked at the code.  Regardless, we'd need to get Vinay Sajep to sign off on any changes here.

Reply:
I changed this like this since the SMTP server used in this tests uses [`decode_data=True`](https://github.com/segevfiner/cpython/blob/f1d84c015faf2d26c0b5f76a20a6af246cb919fe/Lib/test/test_logging.py#L740-L741) which makes it not support `8BITMIME`. See [`smtpd.SMTPServer`'s documentation](https://docs.python.org/3/library/smtpd.html?highlight=smtpd#smtpd.SMTPServer). What's the correct fix for this is open for discussion.

Reply:
Ah, OK, then not using decode_data=True would be the right fix.  decode_data=True is deprecated.